src/mavsdk/plugins/action/action_impl.cpp

```diff
@@ -455,6 +455,7 @@ void ActionImpl::goto_location_async(
             command.params.x = int32_t(std::round(latitude_deg * 1e7));
             command.params.y = int32_t(std::round(longitude_deg * 1e7));
             command.params.maybe_z = altitude_amsl_m;
+            command.params.maybe_param2 = MAV_DO_REPOSITION_FLAGS_CHANGE_MODE;
```

Comment:
I think Windows complains because `MAV_DO_REPOSITION_FLAGS_CHANGE_MODE` is an int, and the param expects a float.

Maybe a static cast does it?

```suggestion
            command.params.maybe_param2 = static_cast<float>(MAV_DO_REPOSITION_FLAGS_CHANGE_MODE);
```

Reply:
Done!